Sources/SwiftSyntaxBuilder/ConvenienceInitializers.swift

```diff
@@ -192,6 +192,28 @@ extension MemberAccessExpr {
 
 // MARK: - StringLiteralExpr
 
+extension String {
+  /// Replace literal newlines with "\r", "\n".
+  fileprivate func replacingNewlines() -> String {
+    var result = ""
+    var input = self[...]
+    while let firstNewline = input.firstIndex(where: { $0.isNewline }) {
+      result += input[..<firstNewline]
+      if input[firstNewline] == "\r" {
+        result += "\\r"
+      } else if input[firstNewline] == "\r\n" {
+        result += "\\r\\n"
+      } else {
+        result += "\\n"
```

Comment:
Replacing is probably fair enough, though you could also do `result += input[firstNewLine].unicodeScalars.map({ $0.escaped(asASCII: true) }).joined())` 🤷